plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/complete-onboarding-wizard.spec.js

```diff
@@ -27,6 +27,10 @@ test.describe( 'Store owner can complete onboarding wizard', () => {
 			storeDetails.us.expectedIndustries
 		);
 		await page.click( 'button >> text=Continue' );
+		await expect( page ).toHaveURL( /.*step=product-types/ );
+		await expect(
+			page.locator( '.product-types button >> text=Continue' )
+		).toBeVisible();
```

Comment:
These assertions make sure that the network calls triggered by clicking the "Continue" button are completed before moving on to the next test.